source/mir/math/sum.d

```diff
@@ -834,31 +873,55 @@ public:
                 {
                     auto s_re = s.re;
                     auto x_re = x.re;
-                    s = x_re + s.im * 1fi;
-                    x = s_re + x.im * 1fi;
+                    static if (is(F : cfloat)) {
+                        s = x_re + s.im * 1fi;
+                        x = s_re + x.im * 1fi;
+                    } else {
+                        import std.complex: Complex;
+                        s = Complex!float(x_re, s.im);
```

Comment:
No need to import Phobos. Something like `F(x_re, s.im)` or `T(x_re, s.im)` should work.

Reply:
The implementation shouldn't import std.complex, tests are can